app/src/main/java/com/instacart/android/challenges/network/NetworkApi.kt

```diff
@@ -25,6 +16,15 @@ interface NetworkApi {
     @GET("order/{order_id}")
     suspend fun fetchOrderByIdCoroutine(@Path("order_id") id: Long): OrderResponse
 
+    @GET("orders")
+    fun fetchOrders(): Call<OrdersResponse>
+
+    @GET("orders")
+    fun fetchOrdersObservable(): Observable<OrdersResponse>
+
+    @GET("orders")
+    suspend fun fetchOrdersCoroutine(): OrdersResponse
+
```

Comment:
I reordered these, since the first step is to use `fetchOrderById` and the second is `fetchOrders`. I would often see people get tripped up on this and be looking at the methods at the top of the file first instead of the appropriate one.